utils/utils.go

```diff
@@ -119,3 +119,10 @@ func splitEnvVar(envVar string) (key, value string) {
 	}
 	return split[0], strings.Join(split[1:], "=")
 }
+
+// This is a general error message for the CLI commands.
+// Because of how command parsing is handled, improperly specified flags may be misinterpreted as arguments.
+// Therefore, these flags will not go through the command's flags verifications, and will not be caught as incorrect flags.
+func GetCliTooManyArgsErrorMessage(numberOfArguments int) string {
+	return fmt.Sprintf("Too many arguments provided (%d in total).\nSome flags may be incorrectly specified, causing them to be misinterpreted as arguments and ignored. Please verify that all flags are valid.", numberOfArguments)
```

Comment:
should we indicate why the flags are incorrectly specified? Maybe add a hint about moving flags before args?

Reply:
@attiasas We can see the correct way to use this through --help.
The issue is that we construct the command in a different way than the common way to do so.
Instead of putting args and then flags we expect the opposite.
So there is no hint to add since its already specified in --help and also- it is just a suggestion. There could be other reasons for this kind of error so I just added a general suggestion and not specific instructions